azurelinuxagent/common/protocol/wire.py

```diff
@@ -1414,6 +1414,12 @@ def parse_plugin(self, plugin):
         ext_handler.properties.upgradeGuid = getattrib(plugin, "upgradeGuid")
         if not ext_handler.properties.upgradeGuid:
             ext_handler.properties.upgradeGuid = None
+
+        try:
+            ext_handler.properties.dependencyLevel = int(getattrib(plugin, "dependencyLevel"))
+        except ValueError:
+            ext_handler.properties.dependencyLevel = None
```

Comment:
Is there a need to propagte None?  If we just set it to a low value (0 or -1 or ...) we can lose the None checks in other parts of the code, right?

Reply:
None gets treated as 0 anyways, so I don't see why not.